### PR TITLE
Fix benchmark script

### LINKED ISSUE #153: Benchmark script is broken
Running the benchmark script produces `0.00` for everything.

### PR DESCRIPTION
**Pull request by zamotany**
Fixes #153

### CODE REVIEW

File: benchmark/run.js [hunk 1]
```diff
@@ -12,17 +12,18 @@ const resultsTable = new Table({
   colWidths: [60, 10, 10],
 });
 
-const outDir = path.join(__dirname, 'suites/_temp');
+const outDir = 'benchmark/_temp';
```
Comment (satya164) [inline benchmark/run.js]: maybe `path.join(__dirname, '_temp');` ?

**Comment by satya164** [comment]
Maybe we should add a snapshot test for the benchmark.
- When someone makes changes, they'll be aware of the perf impact
- Will make sure that the script keeps working

File: benchmark/run.js (hunk 1 above)
Comment (zamotany) [reply benchmark/run.js]: `outDir` must be __relative__ to CWD

**Comment by zamotany** [comment]
Adding snapshot test doesn't make any sense, those are timings which __changes__ between runs even on the same machine. Manually storing results of benchmark in JSON files and comparing it with some threshold would be a acceptable solution.

If you want to ensure the benchmark is working, we can run it on Circle CI.

**Pull request merged**